cla-backend-go/v2/cla_manager/handlers.go

```diff
@@ -125,3 +166,49 @@ func errorResponse(err error) *models.ErrorResponse {
 
 	return &e
 }
+
+func isUserAuthorizedForProjectOrganization(user *auth.User, externalProjectID, externalCompanyID string) bool {
+	if !user.Allowed || !user.IsUserAuthorizedByProject(externalProjectID, externalCompanyID) {
+		return false
+	}
+	return true
+}
+
+// buildErrorMessage helper function to build an error message
+func buildErrorMessage(errPrefix string, params cla_manager.CreateCLAManagerParams, err error) string {
+	return fmt.Sprintf("%s - problem creating new CLA Manager Request using company ID: %s, project ID: %s, first name: %s, last name: %s, user email: %s, error: %+v",
+		errPrefix, params.CompanyID, params.ProjectID, params.Body.FirstName, params.Body.LastName, params.Body.UserEmail, err)
+}
+
+func sendEmailToUserWithNoLFID(projectModel *v1Models.Project, managerEmail string, designeeName string, designeeEmail string) {
+	projectName := projectModel.ProjectName
+	lfidURL := "https://identity.linuxfoundation.org/"
+
+	// subject string, body string, recipients []string
+	subject := fmt.Sprint("CLA: Invitation to create LFID and complete process of becoming CLA Manager")
+	recipients := []string{designeeEmail}
+	body := fmt.Sprintf(`
+<html>
+<head>
+<style>
+body {{font-family: Arial, Helvetica, sans-serif; font-size: 1.2em;}}
+</style>
+</head>
+<body>
+<p>Hello %s,</p>
+<p>User %s was trying to add you as a CLA Manager for Project %s in the EasyCLA system </p>
+<p>In order to become CLA Manager, you will need to create a LFID </p>
+<p>Please create a LFID by following this link %s and let the user <emailid> know your new LFID %s </p>
+<p>Then user %s will be able to add you as a CLA Manager.</p>
+<p>Thanks,
```

Comment:
Include the help line from all our other emails.

See: https://docs.google.com/document/d/1jFlSq1tx66LbBwGbSfJm3QS4st68GblfzBEMZRya9gk/edit#